src/Ninject.Web.AspNetCore.Test/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://go.microsoft.com/fwlink/?LinkID=208121. 
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This should no longer be needed when the tests can be run with `dotnet test ...`

Reply:
Deleted it